wikithingsdb/fetch.py

```diff
@@ -0,0 +1,72 @@
+from wikithingsdb.engine import engine
+from wikithingsdb.models import Page, Redirect, WikiClass, Type, DbpediaClass
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import func
+
+session = sessionmaker(bind=engine)()
+
+
+# def _clean_query(query):
+#     query = query.replace(' ', '_')
+#     query = query.lower()
+#     return query
+
+
+# What are all the ways to refer to ARTICLE?
+def types_of_article(article):
+    # article = _clean_query(article)
+    result = session.query(Page).\
+        join(Page.types).\
+        filter(Page.page_title == article).one()
+    return [x.type for x in result.types]
+
+
+def classes_of_article(article):
+    result = session.query(Page).\
+        join(Page.classes).\
+        filter(Page.page_title == article).one()
```

Comment:
I can make it raise a KeyError with a custom message instead. This will make it easier for other programs to try/except it without the need to import sqlalchemy's obscure exception (I did a similar thing in WikiMap). I'll include comments/documentation about it.